xa/java/helmcharts/transfer-jms/values.yaml

```diff
@@ -0,0 +1,72 @@
+# licensor hereunder covering either the unmodified Software as contributed to or provided by such licensor, or (ii) the Larger Works (as defined below), to deal in both
+# ** (a) the Software, and (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if one is included with the Software (each a "Larger Work" to which
+# the Software is contributed by such licensors), **
+# without restriction, including without limitation the rights to copy, create derivative works of, display, perform, and distribute the Software and make, use, sell,
+# offer for sale, import, export, have made, and have sold the Software and the Larger Work(s), and to sublicense the foregoing rights on either these or other terms. **
+
+# This license is subject to the following condition: The above copyright notice and either this complete permission notice or at a minimum a reference to the UPL must be
+# included in all copies or substantial portions of the Software. **
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+# CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+# Default values for sampleapps.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+sampleapps:
+  teller:
+    name: teller
+    host: teller
+    version: v1
+    image: xa-java-teller:1.0
+    imagePullPolicy: IfNotPresent
+    imagePullSecret: regcred
+    gatewayUriPrefix: /transfers
+    rewriteUriPrefix: /transfers
+    destinationHost: teller
+    propagateTraceHeader: false
+  dept1:
+    name: dept1
+    host: dept1
+    version: v1
+    image: department-helidon:1.0
+    imagePullPolicy: IfNotPresent
+    imagePullSecret: regcred
+    gatewayUriPrefix: /dept1
+    rewriteUriPrefix: /accounts
+    destinationHost: dept1
+    # example for connect string:
+    # jdbc:oracle:thin:@tcps://<host>:<port>/<service_name>?wallet_location=Database_Wallet
+    connectString: <Fill in the connect string dept1>
+    databaseUser: <Fill in the database user dept1>
+    databasePassword: <Fill in the database password dept1>    
+    propagateTraceHeader: false
+    resourceManagerId: <Fill the unique identifier assigned to the dept1 database> # if multiple services use same database, make sure to add same resourceManagerId value in all such places
+    jmsTopicName: <Fill in the oracle teq topic name>
+
+  dept2:
+    name: dept2
+    host: dept2
+    version: v1
+    image: department-spring:1.0
+    imagePullPolicy: IfNotPresent
+    imagePullSecret: regcred
+    gatewayUriPrefix: /dept2
+    rewriteUriPrefix: /accounts
+    destinationHost: dept2
+    sessionAffinity: false
```

Comment:
sessionAffinity is not needed for dept1 ? 
Or this is just to show how to configure sessionAffinity , if yes, then ignore this comment